repository_level_conventions.md

```diff
@@ -9,55 +13,59 @@
 > The fields in the gold datasets should be standardized.
 > Naming conventions - batches: `repoName-issueNumber(-identifier).txt`
 
+
 ## Data Formatting and Precision Conventions
+
 ### Time Point Notation
+
 > [!IMPORTANT]
 > `hh:mm:ss.mmm` with a **DOT**  
 
-The time format for all (gold) datasets in this repository is [ISO 8601 Time Format](https://en.wikipedia.org/wiki/ISO_8601#Times). 
-Specifically, with time precision of up to hours and down to milliseconds placed after seconds with a **DOT**, where `00:00:00.000` means the very beginning of a time-based media (not midnight real-time).  
-This instruction is specifically to be  required for the "gold" data formats throughout this repository, except where exceptions are required and documented.  
-(_TODO: Current/Old gold datasets and tools have yet to be converted._)  
-During raw annotation however, the tool environments may use different time formats because of non-in-house authors. 
-Moving forward, the expectation for any in-house tools and apps are to use this standard. 
-If configurable, the annotation tool should be configured to use this format. If possible, the annotator should also be instructed to use this format.  
+The time format for all (gold) datasets in this repository is [ISO 8601 Time Format](https://en.wikipedia.org/wiki/ISO_8601#Times), with time precision of up to hours and down to milliseconds placed after seconds with a **DOT**, where `00:00:00.000` means the very beginning of a time-based media (not midnight real-time). This instruction is for the "gold" data formats throughout this repository, except where exceptions are required and documented.
+
+> _TODO: Current/Old gold datasets and tools have yet to be converted._
+
+During raw annotation however, third-party annotation tools may use different time formats. The expectation for any in-house tools and apps is to use this standard. If configurable, annotations tool should be configured to use this format. If possible, the annotator should also be instructed to use this format.
+
+> _TODO: the following prose is unclear_
+
 For MPEG-based video files, frame numbers are converted to milliseconds with loss of precision past 3-digits. 
 However, due to exact time -> still-image-fetching being dependent on the video compression/codec/player, there is no expected need for precision past 3-digits. 
 It is assumed that different video players will regenerate images on screen slightly differently based on the decompression algorithms. 
```

Comment:
I guess, these two lines could be something like 
"However, due to algorithmic differences in compression/decompression and their implementations in video players for human watchers, losing temporal precision below milliseconds mustn't be a significant problem"